connection.go

```diff
@@ -40,7 +40,7 @@ func (ac *alisaConn) Close() error {
 // Exec implements database/sql/driver.Execer.
 // Note: result is always nil
 func (ac *alisaConn) Exec(query string, args []driver.Value) (driver.Result, error) {
-	_, err := ac.ali.exec(query)
+	err := ac.ali.exec(query)
```

Comment:
How about directly calll `run`?

Reply:
Do you mean just remove `query` & `exec` and export `Run`?

Reply:
Can call `run` here it's simple enough, and remove `query` and `exec` since they are already exported functions. And `run` does not to be an exported function.